src/function/scalar/struct/struct_extract.cpp

```diff
@@ -87,8 +87,8 @@ static unique_ptr<FunctionData> StructExtractBind(ClientContext &context, Scalar
 	return GetBindData(key_index);
 }
 
-static unique_ptr<FunctionData> StructExtractBindIndex(ClientContext &context, ScalarFunction &bound_function,
-                                                       vector<unique_ptr<Expression>> &arguments) {
+static unique_ptr<FunctionData> StructExtractBindInternal(ClientContext &context, ScalarFunction &bound_function,
+                                                          vector<unique_ptr<Expression>> &arguments, bool by_pos) {
```

Comment:
Minor nit, but this bool should probably be `bool by_name`

Reply:
both functions actually do 'by_pos'. 

I renamed the bool 'struct_extract' now to distinguish between the behavior of 'struct_extract()' and 'struct_extract_at()' => the former only allows positional access on unnamed structs.